index.js

```diff
@@ -33,12 +33,21 @@ class BuildReactForSyncPlugin {
 
   buildReactApp() {
     return this.getStackOutputs().then(outputs => {
+      const command = [];
       const env = { ...process.env, ...outputs };
-      const build = require.resolve('react-scripts/scripts/build');
-      const result = spawn.sync( 'node', [build], { stdio: 'inherit', env });
+      const buildopts = this.serverless.service.custom['build-cra'];
```

Comment:
For consistency's sake, could we call the custom options "build-create-react-app"?